virttest/tests/unattended_install.py

```diff
@@ -1012,6 +1012,9 @@ def setup_nfs(self):
                  (self.nfs_server, self.nfs_dir, self.nfs_mount))
         process.run(m_cmd, verbose=DEBUG)
 
+        if not os.path.exists(self.image_path):
+            os.mkdir(self.image_path)
```

Comment:
we can have it in `__init__()` that avoids the redundant code in `setup_cdrom()`, `setup_url()` and here

Reply:
I was worry about making regressions so I didn't do that. But you're right, for code quality we should do so, thank you.

Reply:
Unfortunately, it does make a regression (see https://github.com/avocado-framework/avocado-vt/pull/1940), and I guess it was not the only one, so let's revert this approach and use the first one I sent, thanks.